docs/source/pages/Profile-Configuration.rst

```diff
@@ -0,0 +1,201 @@
+
+OctoBot configuration is located in the **user** folder:
+
+
+* **user/config.json** is the global configuration file, mostly used to setup the bot exchanges credentials, interfaces and notification settings.
+* **user/profiles/** contains all the `profiles <Profiles.html>`_ created and imported in your OctoBot.
+
+
+OctoBot's web interface allows to easily edit the configuration, however, it is also possible to manually edit configuration files.
+Please be careful when manually editing them or OctoBot won't be able to read them and wont start.
+Json file are readable and editable using any text editor.
+
+.. code-block::
+
+   ERROR    root <class 'Exception'>: Error when load config
+
+This will appear when a configuration file is not a json valid file.
+
+Profile configuration
+================================
+
+**user/config.json** is the technical configuration file of OctoBot, an example
+is available `on github <https://github.com/Drakkar-Software/OctoBot/blob/master/octobot/config/default_config.json>`_.
+
+When starting OctoBot, if the **user** folder is missing or incomplete, it will automatically be created or
+completed with default values.
+
+Strategies
+--------------------
+
+When the profile is read only, the profile strategy and trading mode description are displayed.
```

Comment:
```suggestion
Default profiles can't be edited, you can duplicate them to be able to customize them. On default profiles strategy and trading mode description are displayed instead of selectors to customize the profile.
```